client/coral-plugin-commentbox/CommentBox.js

```diff
@@ -53,12 +71,12 @@ class CommentBox extends React.Component {
         // Execute postSubmit Hooks
         this.state.hooks.postSubmit.forEach(hook => hook(data));
 
+        notifyForNewCommentStatus(addNotification, postedComment.status);
+
         if (postedComment.status === 'REJECTED') {
-          addNotification('error', lang.t('comment-post-banned-word'));
-          !isReply && setCommentCountCache(commentCountCache);
+          !isReply && setCommentCountCache(assetId, commentCountCache);
         } else if (postedComment.status === 'PREMOD') {
-          addNotification('success', lang.t('comment-post-notif-premod'));
-          !isReply && setCommentCountCache(commentCountCache);
+          !isReply && setCommentCountCache(assetId, commentCountCache);
```

Comment:
Yes! I missed this in the big ol merge with coral-embed-stream refactor. Thanks for catching it!